File: benchmarks/cpp/b1_fibonacci.cpp
#include <cstdlib>
#include <iostream>
long long fib(int n) {
    if (n <= 1) return n;
    return fib(n - 1) + fib(n - 2);
}
int main(int argc, char* argv[]) {
    int n = argc > 1 ? std::atoi(argv[1]) : 45;
    std::cout << fib(n) << "\n";
}

File: benchmarks/go/b1_fibonacci.go
package main
import ("fmt"; "os"; "strconv")
func fib(n int) int64 {
    if n <= 1 { return int64(n) }
    return fib(n-1) + fib(n-2)
}
func main() {
    n := 45
    if len(os.Args) > 1 { n, _ = strconv.Atoi(os.Args[1]) }
    fmt.Println(fib(n))
}
